net/device/IotHub.cs

```diff
@@ -11,6 +11,7 @@
 using System.Text.Json;
 using System.Threading;
 using System.Threading.Tasks;
+using static Microsoft.Azure.Iot.Thief.Device.LoggingConstants;
```

Comment:
This is interesting.  How is a static class different from a namespace, and did you have a reason for choosing it here?

Reply:
A static class is just one you can't instantiate. Referencing it in a using statement like this simply means this code can refer to properties and methods in that class without using the class name.

Reply:
Right, I got that part.  I was wondering why you didn't make a namespace called "Microsoft.Azure.Iot.Thief.Device.LoggingConstants" and put the strings into that namespace.  I just remembered that you can't do this.  (https://stackoverflow.com/questions/2818333/namespace-constant-in-c-sharp)

Reply:
Yep, in .NET everything has to be in a class.